dirsrvtests/tests/suites/backups/backup_test.py

```diff
@@ -64,7 +67,34 @@ def test_missing_backend(topo):
     restore_task.create(properties=task_properties)
     restore_task.wait()
     assert restore_task.get_exit_code() != 0
-    
+
+
+@pytest.mark.bz1851967
+@pytest.mark.ds4112
+@pytest.mark.skipif(ds_is_older('1.4.1'), reason="Not implemented")
+def test_db_home_dir_online_backup(topo):
+    """Test that if the dbhome directory is set causing an online backup to fail,
+    the dblayer_backup function should go to error processing section.
+
+    :id: cfc495d6-2a58-4e4e-aa40-39a15c71f973
+    :setup: Standalone Instance
+    :steps:
+        1. Change the dbhome to directory to eg-/tmp/test
+        2. Perform an online back-up
+        3. Check for the correct errors in the log
+    :expectedresults:
+        1. Success
+        2. Failure
+        3. Success
+    """
+    bdb_ldbmconfig = BDB_LDBMConfig(topo.standalone)
+    dseldif = DSEldif(topo.standalone)
+    topo.standalone.stop(timeout=10)
+    dseldif.replace(bdb_ldbmconfig.dn, 'nsslapd-db-home-directory', '/tmp/test')
```

Comment:
It's not a good idea to hardcode in directory names like this because /tmp/test is something I even use and dump values into. You should look at a mktemp or other way of generating randomised folder names.